### PR TITLE
Add examples/ice-single-port-mode

### PR DESCRIPTION
**Pull request by Sean-Der**
Demonstrate how using the new ICEUDPMux multiple PeerConnections
can be served via a single port.

### CODE REVIEW

File: examples/ice-single-port/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# ice-singe-port
```
Comment (yedpodtrzitko) [inline examples/ice-single-port/README.md:1]:
```suggestion
# ice-single-port
```

here and below

File: examples/ice-single-port/index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<html>
+  <head>
+    <title>ice-restart</title>
```
Comment (TannerGabriel) [inline examples/ice-single-port/index.html:3]: Probably a copy-paste mistake. Needs to be change to `ice-single-port`

**Review by TannerGabriel — CHANGES_REQUESTED** [review]
Great changes!

File: examples/ice-single-port/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+## Instructions
+
+### Download ice-singe-port
+This example requires you to clone the repo since it is serving static HTML.
+
+```
+mkdir -p $GOPATH/src/github.com/pion
+cd $GOPATH/src/github.com/pion
+git clone https://github.com/pion/webrtc.git
+cd webrtc/examples/ice-tcp
```
Comment (digitalix) [inline examples/ice-single-port/README.md:17]:
```suggestion
cd webrtc/examples/ice-single-port
```

File: examples/ice-single-port/index.html (hunk 2 above)
Comment (Sean-Der) [reply examples/ice-single-port/index.html:3]: Fixed! nice catch :)

File: examples/ice-single-port/README.md (hunk 3 above)
Comment (Sean-Der) [reply examples/ice-single-port/README.md:17]: Fixed! Thanks @digitalix :)

File: examples/ice-single-port/README.md (hunk 1 above)
Comment (Sean-Der) [reply examples/ice-single-port/README.md]: :fire: fixed, thanks for catching!

**Pull request merged**